pkg/arvo/sys/zuse.hoon

```diff
@@ -3357,7 +3357,7 @@
         ~/  %en
         |=  txt/@
         ^-  (trel @uxH @ud @ux)
-        =+  [k1=(rsh 5 3 key) k2=(end 5 3 key)]
+        =+  [k1=(rsh 6 3 key) k2=(end 6 3 key)]
```

Comment:
I found a bug! `sivb` was previously unusable; it would split the key in the wrong place and then crash because the key was the wrong size.

Do I need to compile a pill for this? Not sure how that works

Reply:
Good catch! `+crub` uses `+siva` and `+sivc`, but nothing uses `+sivb`. I'd guess it's always been broken.

You'll need to create and commit a new solid pill at `bin/solid.pill`. You can use `sh/update-solid-pill`.

Reply:
Cool, seems like it worked!